chat/input-bar/src/InputBar/InputBar.spec.tsx

```diff
@@ -90,4 +90,10 @@ describe('packages/input-bar', () => {
 
     expect(screen.getByText('beta')).toBeInTheDocument();
   });
+
+  test('Hotkey Indicator is rendered when the prop is set', () => {
+    render(<InputBar shouldRenderHotkeyIndicator />);
+
+    expect(screen.getByTestId('hotkey-indicator')).toBeInTheDocument();
```

Comment:
nit: but might want a more robustly-prefixed id for this (lg-chat-hotkey-indicator) or something similar, so that it does not conflict with anything else on the page.